### ISSUE TITLE
Export/Import of provisioning policies is broken

### ISSUE THREAD

**Issue by Gorxe6** [BlueBayTechnologies/ISIM-Code-manager#1]
If a Provisioning Policy has multiple entitled roles and multiple entitlements, then the exported entries do not maintain these multiple relationships properly.

This project's products are not safe for use.

**Comment by ninovic92** [comment]
Agreed the AL "ExportProvisioningPolicies" only exports one entitlement. You could fix it by adding a FOR-EACH component and adding "Work Attribute Name: erentitlements" and then a loop attribute name of your choice. I export all erentitlements to separate .xml files.

**Comment by alexivkin** [comment]
Thanks for the hint! I will keep this open so others who use it can benefit
